src/pages/DefaultOrgSelector/DefaultOrgSelector.spec.jsx

```diff
@@ -60,12 +73,14 @@ describe('DefaultOrgSelector', () => {
     myOrganizationsData,
     useUserData,
     isValidUser = true,
+    trialStatus = 'NOT_STARTED',
+    planName = 'users-inappm',
```

Comment:
Nitpick: this scenario could never happen, so maybe change this to either `users-basic` or `users-free`